src/main/java/duke/exception/IncompleteCommandError.java

```diff
@@ -0,0 +1,22 @@
+package duke.exception;
+
+public class IncompleteCommandError extends UnknownCommandException {
```

Comment:
Regarding errors vs exceptions \:
https://stackoverflow.com/questions/5813614/what-is-difference-between-errors-and-exceptions

(e.g. stackOverflowError is something that is unreasonable for the application to catch since it depends on the defined  maximum stack size of the platform the application runs on ) ( assuming a well programmed recursive process )